### PR TITLE
kyma alpha deploy pipeline

### PR DESCRIPTION
**Pull request by strekm**
**Description**

Changes proposed in this pull request:

- kyma alpha deploy pipeline running on every PR, using stable kyma cli
- removed kyma alpha upgrade for now

**Related issue(s)**
https://github.com/kyma-project/test-infra/issues/3409

### CODE REVIEW

File: prow/scripts/lib/gardener/aws.sh [hunk 1]
```diff
@@ -84,6 +84,16 @@ gardener::provision_cluster() {
     )
 }
 
+gardener::alpha_deploy_kyma() {
+  log::info "Deploying Kyma"
```
Comment (mjakobczyk) [inline prow/scripts/lib/gardener/aws.sh:88]: For now Kyma deployment works the same for every cluster created through Gardener (AWS, GCP, Azure). Wouldn't it be more readable and maintainable to implement this function in another file and import it when needed for every provider-specific script?

File: prow/scripts/lib/gardener/azure.sh [hunk 2]
```diff
@@ -137,6 +137,16 @@ gardener::provision_cluster() {
     set +x
 }
 
+gardener::alpha_deploy_kyma() {
+  log::info "Deploying Kyma"
```
Comment (mjakobczyk) [inline prow/scripts/lib/gardener/azure.sh:141]: As above.

File: prow/scripts/lib/gardener/gcp.sh [hunk 3]
```diff
@@ -85,6 +85,16 @@ gardener::provision_cluster() {
     )
 }
 
+gardener::alpha_deploy_kyma() {
+  log::info "Deploying Kyma"
```
Comment (mjakobczyk) [inline prow/scripts/lib/gardener/gcp.sh:89]: As above.

File: prow/scripts/lib/gardener/aws.sh (hunk 1 above)
Comment (adamwalach) [reply prow/scripts/lib/gardener/aws.sh:88]: If we keep only one version of this function then I think we should put in https://github.com/kyma-project/test-infra/blob/master/prow/scripts/lib/kyma.sh

Comment (strekm) [reply prow/scripts/lib/gardener/aws.sh:88]: atm idk if we will have only one version or many, i guess it was not tested yet. i have no problem moving it to a separate file then, if we will see a need to have specifics for providers then we can implement it.

**Review by adamwalach — APPROVED** [review]

**Review by superojla — APPROVED** [review]

**Review by mjakobczyk — APPROVED** [review]

**Pull request merged**